src/frontend/package.json

```diff
@@ -11,7 +11,7 @@
     "local-dev": "API_URL=http://backend.genepinet.local:3000 npm run dev --  -p 8000",
     "tests": "jest --config jest/config.js",
     "e2e": "DEBUG=pw:api jest --config jest/e2e.config.js",
-    "e2e-staging-from-local": "AWS_PROFILE=genepi-dev FRONTEND_URL=https://staging.genepi.czi.technology/ DEPLOYMENT_STAGE=staging make e2e",
+    "e2e-staging-from-local": "AWS_PROFILE=genepi-dev FRONTEND_URL=https://staging.genepi.czi.technology DEPLOYMENT_STAGE=staging make e2e",
```

Comment:
Was this change intended?

Reply:
Ah yes that's needed and intended, otherwise URL might get double forward slashes!